lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -185,7 +185,7 @@ class ReportUIFeatures {
 
       filterInput.id = id;
       filterInput.addEventListener('change', e => {
-        // Remove rows from the dom and keep track of them to readd on uncheck.
+        // Remove rows from the dom and keep track of them to re-add on uncheck.
```

Comment:
![image](https://user-images.githubusercontent.com/4071474/58148874-5395e280-7c15-11e9-87a8-4cd58b603b7d.png)

Reply:
*so* close to `tracium` :)